src/pages/AuthorPage.jsx

```diff
@@ -77,7 +79,14 @@ const styles = theme => ({
         '&:hover $authorEditButton': {
             visibility: 'visible'
         }
-    }
+    },
+  actionMenu: {
+      paddingTop: theme.spacing(1),
+      paddingBottom: theme.spacing(1),
+      margin: 0,
+      position: 'absolute',
+      right: (C.COL_WIDTH - C.CARD_COL_WIDTH)/2,
```

Comment:
please replace following 3 lines
```
margin: 0,
position: 'absolute',
right: (C.COL_WIDTH - C.CARD_COL_WIDTH)/2,
 ```
with 
`float:right`
to fix horizontal alignment issue

![image](https://user-images.githubusercontent.com/4512335/86787666-54fdb680-c033-11ea-9328-0f1fab3f3b49.png)